fastlane/Fastfile

```diff
@@ -61,6 +61,58 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases and opens a PR"
+lane :open_pr_upgrading_dependencies do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  
+  new_versions = update_native_dependencies_to_latest()
+  
+  repo_status = Actions.sh("git status --porcelain")
```

Comment:
In the `Fastfile`, you shouldn't need to add the `Actions`. Same in the other sh scripts
```suggestion
  repo_status = sh("git status --porcelain")
```